apps/glusterfs/volume_entry_allocate.go

```diff
@@ -63,6 +64,165 @@ func (v *VolumeEntry) allocBricksInCluster(db *bolt.DB,
 	}
 }
 
+func (v *VolumeEntry) replaceBrickInVolume(db *bolt.DB, executor executors.Executor,
```

Comment:
We can discuss more about this if you want in mail. 
http://lists.gluster.org/pipermail/heketi-devel/2017-February/000064.html